xarray/conventions.py

```diff
@@ -350,6 +350,25 @@ def stackable(dim):
         drop_variables = []
     drop_variables = set(drop_variables)
 
+    # Time bounds coordinates might miss the decoding attributes
+    # https://github.com/pydata/xarray/issues/2565
```

Comment:
Would it also make sense to add a comment with the URL for the ["Cell Boundaries" section](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/cf-conventions.html#cell-boundaries) of the CF conventions here?